src/core/client_config/lb_policies/pick_first.c

```diff
@@ -57,10 +57,8 @@ typedef struct {
 
   /** mutex protecting remaining members */
   gpr_mu mu;
-  /** the selected channel
-      TODO(ctiller): this should be atomically set so we don't
-                     need to take a mutex in the common case */
-  grpc_connected_subchannel *selected;
+  /** the selected channel (a grpc_connected_subchannel) */
```

Comment:
The above comment says mutex protecting the remaining members, but the mutex does not protect "selected" . Maybe  this comment here should say that (e.g., "Note that mu does not protect the field 'selected' " or on the mutex comment itself (protect remaining members, except for atomic field selected)